cg/meta/workflow/raredisease.py

```diff
@@ -149,11 +152,58 @@ def get_managed_variants(self) -> list[str]:
         return self._get_managed_variants(genome_build=GenePanelGenomeBuild.hg19)
 
     def get_workflow_metrics(self, sample_id: str) -> dict:
+        """Return Raredisease workflow metric conditions for a sample."""
         sample: Sample = self.status_db.get_sample_by_internal_id(internal_id=sample_id)
-        metric_conditions: dict[str, dict[str, Any]] = dict(RAREDISEASE_METRIC_CONDITIONS)
-        self.set_order_sex_for_sample(sample, metric_conditions)
+        if "-" not in sample_id:
```

Comment:
Is this check needed? In all of statusdb there are only three samples, all from 2017, whose internal ids contain a dash. Or is the sample id the name?

Reply:
This is not always a sample id. We added this check because we use this function with str like `sample1-sample2`. This is a function overridden from the NfAnalysisAPI so renaming the parameter will be a bit invasive

Reply:
Then I do not agree that we want to use this method? Or why not have two separate methods given that the behaviour differs depending on the input?

Reply:
This function will still be called from a parent class using the pattern as "sample id". I think the best I can think of is doing like:
```python
def get_workflow_metrics(self, sample_id: str) -> dict:
    return self._some_other_function(combined_sample_ids=sample_id)
```
It is worth mentioning that the whole QC parsing is planned to be taken out of the Analysis API in the future